docs/walk-through/loops.md

```diff
@@ -177,6 +178,32 @@ spec:
       args: ["echo sleeping for {{inputs.parameters.seconds}} seconds; sleep {{inputs.parameters.seconds}}; echo done"]
 ```
 
+## `withSequence` example
+
+This example run a simple template multiple times using `withSequence`.
+
+```yaml
+apiVersion: argoproj.io/v1alpha1
+kind: Workflow
+metadata:
+  generateName: loop-sequence-
+spec:
+  entrypoint: loop-sequence-example
+
+  templates:
+  - name: loop-sequence-example
+    steps:
+    - - name: run-pod
+        template: run-pod
+        withSequence:
+          count: "5"
+
+  - name: run-pod
+    container:
+      image: argoproj/argosay:v2
+      args: [sleep, 1s]
```

Comment:
this should probably do something other than sleep as an example. the output should confirm that the sequence is running. for instance have it print "hello world". can use `whalesay` like the `withItems` example

Reply:
Follow-up to use `whalesay` for consistency: https://github.com/argoproj/argo-workflows/pull/12577#discussion_r1498185891